waku/common/wakubridge.nim

```diff
@@ -91,6 +91,7 @@ func toWakuMessage(env: Envelope): WakuMessage =
   # Translate a Waku v1 envelope to a Waku v2 message
   WakuMessage(payload: env.data,
               contentTopic: toV2ContentTopic(env.topic),
+              timestamp: float64(env.expiry - env.ttl),
```

Comment:
I guess ttl is a fixed value and does not change by the routing peers, right?

Reply:
It is prudent to check whether the expiry is larger than ttl, otherwise, it results in a very large timestamp (instead of a negative value).  Although it might be a concern at the v1 level, not v2.